openbas-front/src/admin/components/common/injects/Injects.js

```diff
@@ -175,7 +175,12 @@ const Injects = (props) => {
   const classes = useStyles();
   const { t, tPick } = useFormatter();
   const [selectedInjectId, setSelectedInjectId] = useState(null);
-  const [showTimeline, _setShowTimeline] = useState(true);
+  const [showTimeline, setShowTimeline] = useState(
+    () => {
+      const storedValue = localStorage.getItem(exerciseOrScenarioId);
```

Comment:
we need a more specific key to understand what is store in the localstorage
${exerciseOrScenarioId}_injects_timeline or something else